Helix Edior - nimlangserver exits on startup

I've just started programming nim and wanted to use nimlangserver in my helix editor.

Installed it with `nimble install nimlangserver` and tested it with both the version 1.6.1 and version 2.0.0.

But the language-server always exits on startup of helix with the following error:

```
2023-11-09T16:16:45.888 helix_lsp::transport [ERROR] nimlangserver err <- "DBG Router: dispatching                        method=initialize id=0\n"
2023-11-09T16:16:45.888 helix_lsp::transport [ERROR] nimlangserver err <- "DBG Initialize received...                    \n"
2023-11-09T16:16:45.888 helix_lsp::transport [ERROR] nimlangserver err <- "DBG Router: sending response                   method=initialize id=0\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "DBG Router: dispatching                        method=initialized id=null\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "DBG Client initialized.                       \n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "DBG Requesting configuration from the client  \n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "DBG Router: dispatching                        method=textDocument/didOpen id=null\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "DBG New document opened for URI:               uri=file:///home/axelr/test.nim\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "Shutting down due to an error: Invalid scheme: , only \"file\" is supported\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "utils.nim(87)            uriToPath\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "asyncfutures.nim(389)    read\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "asyncfutures.nim(389)    read\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "asyncfutures.nim(389)    read\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "???                      routeIter\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "asyncfutures.nim(389)    read\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "asyncfutures.nim(389)    read\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err <- "\n"
2023-11-09T16:16:45.889 helix_lsp::transport [ERROR] nimlangserver err: <- StreamClosed
```

So I think, this is the problem: `Shutting down due to an error: Invalid scheme: , only \"file\" is supported`
But I don't know, how to solve it.

Can anyone help out? I've made no other changes for the nim lang in the `config.toml` of helix.

Thanks a lot.

--- a/nimlangserver.nim
+++ b/nimlangserver.nim
@@ -177,7 +177,10 @@ proc getWorkspaceConfiguration(ls: LanguageServer): Future[NlsConfig] {.async.} 
 
 proc getRootPath(ip: InitializeParams): string =
   if ip.rootUri.isNone or ip.rootUri.get == "":
-    return getCurrentDir().pathToUri.uriToPath
+    if ip.rootPath.isSome and ip.rootPath.get != "":
+      return ip.rootPath.get
+    else:
+      return getCurrentDir().pathToUri.uriToPath
   return ip.rootUri.get.uriToPath
 
 proc getProjectFile(fileUri: string, ls: LanguageServer): Future[string] {.async.} =

